src/main/java/teammates/logic/api/Logic.java

```diff
@@ -1387,4 +1387,8 @@ public int getNumOfGeneratedChoicesForParticipantType(String courseId, FeedbackP
         return feedbackQuestionsLogic.getNumOfGeneratedChoicesForParticipantType(courseId, generateOptionsFor);
     }
 
+    public int getSubmittedResponseCountForQuestion(String questionId) {
```

Comment:
I have some doubt over the granularity of the metadata fetch:
* Is per question response count really useful? (maybe one could just use the question result call to get all the response and count the length in an inexpensive way)
* I think you could actually make it course/session wide to improve the usefulness per call.